modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java

```diff
@@ -1564,6 +1566,72 @@ public void testSearchObjectEntries() throws Exception {
 		Assert.assertEquals(0, baseModelSearchResult.getLength());
 	}
 
+	@Test
+	public void testSystemObjectExtendedProperties() throws Exception {
```

Comment:
An idea: what do you think of putting "Methods" as suffix?

Reply:
Done!

Reply:
Great. Thanks. 🙏🏻